### DISCUSSION TITLE
kubernetes/2021/12/31/welcome-to-jekyll

### DISCUSSION THREAD

**Discussion by giscus** [codeflitting/codeflitting.github.io#2] [Announcements]
# kubernetes/2021/12/31/welcome-to-jekyll

You’ll find this post in your _posts directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run jekyll serve, which launches a web server and auto-regenerates your site when a file is updated. Jekyll requires blog post files to be named according to the following format:

http://localhost:4000/kubernetes/2021/12/31/welcome-to-jekyll.html

**Comment by codeflitting** [comment]
sdddddddddddddddddd
